app/views/documents/form_partials/_privacy.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form.label :numbers, "Document number(s)", class: "govuk-label", for: "numbers" %>
+
+<div id="event-name-hint" class="govuk-hint">
+  Where documents contain multiple drawings, enter multiple numbers separated by a comma (e.g. "25A-V2, 25B-V2").
+</div>
+<div class="numbers">
+  <%= form.text_area :numbers, class: "govuk-input govuk-input--width-20", id: "numbers" %>
+</div>
+
+<div class="display">
+  <%= form.govuk_radio_buttons_fieldset(:status, legend: { size: 's', text: 'Do you want to list this document on the decision notice?' }) do %>
+    <%= form.govuk_radio_button :referenced_in_decision_notice, 'true', label: { text: 'Yes' } %>
+    <%= form.govuk_radio_button :referenced_in_decision_notice, 'false', label: { text: 'No' } %>
+  <% end %>
+</div>
+
+<div class="publish">
+  <%= form.govuk_radio_buttons_fieldset(:status, legend: { size: 's', text: 'Should this document be made publicly available?' }) do %>
+    <div id="event-name-hint" class="govuk-hint">
+      Please indicate whether this document should be made available for public viewing on the planning register or via the public API. Select "No" if the document contains any sensitive personal information
```

Comment:
![image](https://user-images.githubusercontent.com/9452321/109823068-7e58fb80-7c2f-11eb-99fa-889408b6f353.png)
I think we need a little space here